Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/special.yml

```diff
@@ -48,10 +48,26 @@
 - type: entity
   id: BulletSpecialMindbreaker
   name: bullet (.38 special mindbreaker)
-  parent: BaseBullet
+  parent: BaseBulletPractice
   noSpawn: true
   components:
   - type: Projectile
     damage:
       types:
-        Blunt: 3
+        Blunt: 1
+  - type: SolutionContainerManager
+    solutions:
+      ammo:
+        maxVol: 9
+  - type: RefillableSolution
+    solution: ammo
+  - type: DrainableSolution
```

Comment:
Adding drainable solution here would actually catapult the .38 special rounds into "Genuinely broken" territory. Since it'll let you take a syringe and remove the mindbreaker from individual bullets, and replace them with whatever you want. Any kind of toxin comes to mind, but most especially Amatoxin and Lexorin, both immediately fatal with >7 units.

Reply:
> Adding drainable solution here would actually catapult the .38 special rounds into "Genuinely broken" territory. Since it'll let you take a syringe and remove the soulbreaker from individual bullets, and replace them with whatever you want. Any kind of toxin comes to mind, but most especially Amatoxin and Lexorin, both immediately fatal with >7 units.

it would be funny

Reply:
no thats syringe gun ss13

Reply:
Good luck using a syringe on the bullet coming at you.
I just copied whatever the tranquilizer pellets does
https://github.com/DeltaV-Station/Delta-v/blob/e1337f7f5ec87a4687a995f8707fc421c575cbad/Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml#L91-L116

Reply:
OH well that explains why Tranquilizer shells currently don't work. They're straight up empty. 
![image](https://github.com/DeltaV-Station/Delta-v/assets/16548818/53a5f237-b162-4978-845c-487c6c91d0e1)

Reply:
Tranquilizer shell works perfectly fine
![ss+(2024-02-10+at+06 05 11)](https://github.com/DeltaV-Station/Delta-v/assets/9823203/7b4d3eba-10f0-44b9-889e-8c7f40b7118e)

Reply:
![image](https://github.com/DeltaV-Station/Delta-v/assets/16548818/e8da2264-527e-49b0-85d6-eac4916a7403)
Oh I am mistaken then, carry on! Okay so totally ignore my review about the solution container manager thing, I haven't fully woken up yet, I just realized its a container for ammo and not a container for drugs.

Reply:
It's a bit weird how it is but yes, you probably could use a syringe on the actual projectile but it'll be far gone before the doafter ever finishes.
You cannot syringe the cartridge itself so it's fine.